scripts/quota_log.py

```diff
@@ -80,16 +92,32 @@ def save_results(disk_quota_data, cpu_hours_data, db_config):
 
     couch = couchdb.Server(server)
     db = couch['uppmax']
+    print 'Connection established'
 
     # save results in the database
     for project in merged_results.values():
-        db.save(project)
+        project_id, project_rev = db.save(project)
+        print 'Project:' +  project_id, project['project'] + ' has been updated'
 
 
 
 if __name__ == "__main__":
-    with open('/home/funk_001/.ngi_config/statusdb.yaml', 'r') as config_file:
-        config = yaml.load(config_file.readall())
+
+    # parse arguments from command line
+    parser = argparse.ArgumentParser(description="Formats uquota \
+        information as a dict, and sends it to a given CouchDB.")
+
+    parser.add_argument("--config", dest="config", default="", \
```

Comment:
so you set default to empty string, but then you declare a default ? is that a bit redundant ?